docs/cookbook/ubuntu-setup.md

```diff
@@ -0,0 +1,232 @@
+# Ubuntu hosting setup
```

Comment:
Unfortunately a few steps in git forgot this was changing the original guide page.